json_inttypes.h

```diff
@@ -2,7 +2,7 @@
 #ifndef _fj_json_inttypes_h_
 #define _fj_json_inttypes_h_
 
-#include "json_config.h"
+#include "config.h"
```

Comment:
since json_inttypes.h references config.h, and json_inttypes.h is installed to /usr/include, config.h will need to be installed as well, and in this case, naming it json_config.h seems more appropriate.